android/tangram/jni/platform_android.cpp

```diff
@@ -328,5 +337,11 @@ void initGLExtensions() {
     glGenVertexArraysOESEXT = (PFNGLGENVERTEXARRAYSOESPROC) dlsym(libhandle, "glGenVertexArraysOES");
 }
 
+std::string stringFromJString(JNIEnv* jniEnv, jstring string) {
+    const char* cstring = jniEnv->GetStringUTFChars(string, NULL);
```

Comment:
Oh, I see! How lovely that JNI has so many ways to do the same thing :P

I guess I'm not especially worried about forgetting to release resources since this is encapsulated in a reusable function, I can change it if you have a strong preference.